projects/packages/waf/src/class-waf-runtime.php

```diff
@@ -69,16 +80,23 @@ class Waf_Runtime {
 	/**
 	 * Transforms.
 	 *
-	 * @var Waf_Transforms[]
+	 * @var Waf_Transforms
```

Comment:
Update the type here and below in the next one - these variables aren't arrays, they are single instances of the class.